packages/webapp/src/containers/AddCropVariety/AddCropVariety.jsx

```diff
@@ -31,8 +31,8 @@ function AddCropVarietyForm({ history, match }) {
       ...persistedFormData,
       ...data,
       crop_variety_name: data.crop_variety_name.trim(),
-      crop_varietal: data.crop_varietal.trim(),
-      crop_cultivar: data.crop_cultivar.trim(),
+      crop_varietal: data.crop_varietal.length ? data.crop_varietal.trim() : null,
+      crop_cultivar: data.crop_cultivar.length ? data.crop_cultivar.trim() : null,
```

Comment:
I noticed that for some reason the fields are never set to null in the database unless there is a check to see if they are empty and then their values set explicitly

Reply:
as I understand you considered the minLength check to not be helpful

Reply:
Sorry if I communicated this badly, yeah just seems weird to allow null and set a min length to 1 at the same time. This could probably be investigated at some point in the future.